rest/taskana-rest-spring/src/test/java/pro/taskana/rest/resource/assembler/ClassificationResourceAssemblerTest.java

```diff
@@ -0,0 +1,125 @@
+package pro.taskana.rest.resource.assembler;
+
+import java.time.Instant;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+
+import pro.taskana.Classification;
+import pro.taskana.ClassificationService;
+import pro.taskana.exceptions.ClassificationAlreadyExistException;
+import pro.taskana.exceptions.ClassificationNotFoundException;
+import pro.taskana.exceptions.ConcurrencyException;
+import pro.taskana.exceptions.DomainNotFoundException;
+import pro.taskana.exceptions.InvalidArgumentException;
+import pro.taskana.exceptions.NotAuthorizedException;
+import pro.taskana.impl.ClassificationImpl;
+import pro.taskana.rest.TestConfiguration;
+import pro.taskana.rest.resource.ClassificationResource;
+
+/**
+ * Test for {@link ClassificationResourceAssembler}.
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = {TestConfiguration.class})
+@WebAppConfiguration
```

Comment:
a web context is too much for this simple test. you even don't need spring at all and just instantiate the beans yourself..